src/PostalServiceProvider.php

```diff
@@ -36,12 +37,16 @@ public function boot()
      */
     public function register()
     {
+        $this->app->afterResolving(TransportManager::class, function (TransportManager $manager) {
+            $this->extendForPostal($manager);
+        });
+
         $this->app->afterResolving(MailManager::class, function (MailManager $manager) {
-            $this->extendMailManager($manager);
+            $this->extendForPostal($manager);
         });
     }
 
-    public function extendMailManager(MailManager $manager)
+    public function extendForPostal($manager)
```

Comment:
we can't change our public signatures when releasing a minor version, we should keep this as `extendMailManager` (even though it can also extend `TransportManager`) for the time being